client/container.go

```diff
@@ -620,19 +610,21 @@ func (x *PrmAnnounceSpace) SetValues(vs []container.SizeEstimation) {
 //
 // Context is required and must not be nil. It is used for network communication.
 //
+// Announcements parameter MUST NOT be empty slice.
+//
 // Return errors:
 //   - [ErrMissingAnnouncements]
-func (c *Client) ContainerAnnounceUsedSpace(ctx context.Context, prm PrmAnnounceSpace) error {
+func (c *Client) ContainerAnnounceUsedSpace(ctx context.Context, announcements []container.SizeEstimation, prm PrmAnnounceSpaceOptional) error {
 	// check parameters
 	switch {
-	case len(prm.announcements) == 0:
+	case len(announcements) == 0:
```

Comment:
Just `if` here? `switch is some excessive now.

Reply:
Sure, replaced with if